main.cpp

```diff
@@ -78,44 +86,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
-
+ */
+ bool rentAHouse(int houseType, bool job = true)
+ {
+    ignoreUnused(houseType, job);
+    return {};
+ }

```

Comment:
This function is not lined up with the Indentation guidelines in Replit.
Align your code with the vertical lines
- [ ] revise